.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Not everyone wants to use nix. We don't want to force nix on people that don't want to use it.